tools/releaseBuild/vstsbuild.ps1

```diff
@@ -74,7 +108,12 @@ End {
         Import-Module "$location/dockerBasedBuild" -Force
         Clear-VstsTaskState
 
-        Invoke-Build -RepoPath $resolvedRepoRoot  -BuildJsonPath './tools/releaseBuild/build.json' -Name $Name -Parameters $PSBoundParameters
+        $buildParameters = @{
+            ReleaseTag = $ReleaseTag
```

Comment:
Previously, you pass in `$PSBoundParameters`, so if `-ReleaseTag` is not specified, it won't be passed to `Invoke-Build`.
Now you always pass in `ReleaseTag`, and the value will be `$null` in case `-ReleaseTag` is not specified when calling this script. Will this cause any problems?

Reply:
The new hashtable I have has `ReleaseTag`

Reply:
No, this won't cause any issues.  I've verified this previously.